modules/virt-installing-a-watchdog.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog_{context}"]
+= Installing a watchdog device
+
+To activate a watchdog device, install the `watchdog` package on your virtual machine and start the watchdog service.
+
+.Procedure
+
+. Install the watchdog package and dependencies:
++
+[source,terminal]
+----
+# yum install watchdog
```

Comment:
Are you sure that the user does not need to be root or have root access?

Did QE check these commands? 

I think this would be either `$ yum install watchdog` or `# sudo yum install watchdog`.